src/ui/flutter_app/lib/services/audios.dart

```diff
@@ -60,12 +61,12 @@ class Audios {
 
   static Future<void> loadSounds() async {
     if (Platform.isWindows) {
-      debugPrint("$_tag Audio Player does not support Windows.");
+      debugPrint("$_tag Sound pool does not support Windows.");
       return;
     }
 
     if (_initialized) {
-      debugPrint("$_tag Audio Player is already initialized.");
+      debugPrint("$_tag Sound pool is already initialized.");
```

Comment:
also why did you change the logs? The thing already being initialized is sound pool not audio player. please change them back